spec/javascripts/reducers/notification_spec.js

```diff
@@ -0,0 +1,38 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/notifications";
+
+describe("Notification Reducer",() => {
+  it("should return initialState", () => {
```

Comment:
I don't know what is the pattern for the test's messages but I would remove the word "should" and use `describe` for each scenario and change the `it` message for what is the returning.

####  The word "should"

> used to indicate what is probable.
As the test result is certain and not likely I would not use "should".

#### Description

I would probably do something like this:

```js
  describe('DEFAULT', () => {
    it('returns an empty array', () => {
      expect(reducer(undefined, [])).toEqual([]);
    });
  })
  

  describe('ADD_NOTIFICATION', () => {
    it('adds a new notification into state', () => {
      const state = [{ id: 1 }, { id: 2 }];

      const action = {
        type: actionsTypes.ADD_NOTIFICATION,
        notification: { id: 3 }
      };
      
      // Also, I would recommend to you to write the whole expected object to keep your test more explicit.
      expect(reducer(state, action)).toEqual([{ id: 1 }, { id: 2 }, { id: 3 }]);
    });
  });
```

Reply:
thanks @nolleto , changing right now. If you have any articles or tips for writing better tests, I would be glad to read :D